### ISSUE TITLE
Next version 3.0.0

### ISSUE THREAD

**Issue by f3l1x** [o5/grido#288]
Hi. 

Thanks you for great job. I have a question, when do you plan for next release? With nette 2.4 support. We have a few projects waiting for that. :-)

Thanks.

**Comment by JosefRypacek** [comment]
Hi,

grido is working with nette 2.4, but you need to use newest version (branch **master**)
If you are using composer and/or bower, you have to set dependencies as:
```
composer: "o5/grido": "dev-master"
bower: "grido": "#611ae6b"
```

**Comment by o5** [comment]
@f3l1x I would like to release version **3.0** in this year.

**Comment by lukasojd** [comment]
+1

**Comment by f3l1x** [comment]
@o5 How is it going?

**Comment by f3l1x** [comment]
@o5 Any progress here? I'm not happy to say it, because I like grido, but this slowly progress kill me. It's look like we have to move to other datagrid .. :( I'm so sorry.

**Comment by o5** [comment]
@f3l1x sorry, I don't time for this project and don't want to promise something like date when next release will be.

**Comment by f3l1x** [comment]
I was thinking about forking this project and develop our own version, because @o5 does not have time for this. Although, what about to choose other maintainer(s) to get this project alive again.

Just an idea.

**Comment by o5** [comment]
@f3l1x: Definitely agree with you! But there are some "but"s :)

I need to prepare development environment with Docker (don't like to install some development tools on local machine), because I would like to "manually" review of pull requests (don't like merge button). I'm not also a big fan with adding other maintainers in current project state.

So my first task should create docker image with php + nginx. I've already created similar images for some companies in which I worked, but of course, it was not open-source. I also tried to use official php image with build-in server, but it was too slow (assets should be definitely served by nginx). There is also official image with Apache, but I used Apache 5years ago and I would like slim image based on Alpine. This docker image should be separated project and it will take some time.

Next task will be integrate it with grido-sandbox, fix Grido for latest Nette, resolve reported issues and release new version (I'm not sure for version **3.0** because there are a lot of things which I'd like to change).

As I already somewhere said, it's primary about motivation :/ I switched from web-based application (serving HTML) to microservices 3years ago and unfortunately I have not a big motivation to giving my time to Nette community (a small local community where haters are more active than fans). Maybe you are motivated to do something for Nette community because you are actively working with Nette but I not.
